lib/SPIRV/SPIRVReader.cpp

```diff
@@ -587,7 +589,8 @@ bool SPIRVToLLVM::isDirectlyTranslatedToOCL(Op OpCode) const {
     // everything except atomics, groups, pipes and media_block_io_intel is
```

Comment:
I suggest to update this comment: right now it tries to explicitly list all exceptions. Probably some more generic phase that "there are some exceptions" would be better since we increase this list

Reply:
Done.